sys-apps/firejail/firejail-0.9.66.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit toolchain-funcs python-single-r1 linux-info
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+	SRC_URI="https://github.com/netblue30/${PN}/releases/download/${PV}/${P}.tar.xz"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/netblue30/firejail.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="Security sandbox for any type of processes"
+HOMEPAGE="https://firejail.wordpress.com/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="X apparmor +chroot contrib +dbusproxy +file-transfer +globalcfg +network +private-home +suid test +userns +whitelist"
+# Needs a lot of work to function within sandbox/portage
+# bug #769731
+RESTRICT="test"
+
+RDEPEND="!sys-apps/firejail-lts
+	apparmor? ( sys-libs/libapparmor )
+	contrib? ( ${PYTHON_DEPS} )
+	dbusproxy? ( sys-apps/xdg-dbus-proxy )"
+
+DEPEND="${RDEPEND}
+	sys-libs/libseccomp
```

Comment:
Hm, indeed, I see no binaries produced by firejail that are linked to libseccomp, nor find any obvious uses of them being dlopen'ed or similar (there is what looks to be a private copy/implementation of some seccomp bits?). Only references to `#include <linux/seccomp.h>` and the like at build time.

I don't have any system that doesn't have libseccomp but does have firejail, to confirm the seccomp bits work fine w/o libseccomp installed. So I'll make one and follow up after...

Reply:
Confirmed, if I `emerge -C libseccomp`, running `firejail sleep 100` still imposes a seccomp filter, which can be read/verified/inspected in another terminal:

```# firejail --seccomp.print=18535
Switching to pid 18536, the first child process inside the sandbox
FILE: /proc/18536/root/run/firejail/mnt/seccomp/seccomp.protocol
 line  OP JT JF    K
=================================
 0000: 20 00 00 00000004   ld  data.architecture
 0001: 15 04 00 c000003e   jeq ARCH_64 0006 (false 0002)
 0002: 20 00 00 00000000   ld  data.syscall-number
...
```

So yeah, oddly enough, we need it only at build time. So does that mean DEPEND= is correct/sufficient?

Reply:
Yeah, DEPEND is right there (because in the cross case e.g. arm, we want it for the "target" (not in the autotools sense, HOST in that terminology, not e.g. amd64 -- hence not BDEPEND, even though both are satisfied "at build time" (in different places)).

Thanks for checking this!